src/services/AbstractDFG.jl

```diff
@@ -61,79 +61,79 @@ end
     $(SIGNATURES)
 Add a DFGVariable to a DFG.
 """
-function addVariable!(dfg::G, variable::DFGVariable)::Bool where G <: AbstractDFG
+function addVariable!(dfg::G, variable::V)::Bool where {G <: AbstractDFG, V <: AbstractDFGVariable}
```

Comment:
Johan correctly pointed out that we should not overdo static parameters:
https://docs.julialang.org/en/v1/manual/style-guide/#Don't-use-unnecessary-static-parameters-1

Reply:
not serious, but keep in mind for future work.  I'm pretty guilty of this myself.